### PR TITLE
Bea-Haydee skeleton blog post

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

**Comment by hsvab** [comment]
No suggestions on the text, it is ok.

File: content/blog/2023-05-23-champions-program-teams/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+</br>
+
+## Beatriz Milz - Mentor
+
+{{< figure src = "/img/team/beatriz-milz-mentor.jpeg" class = "pull-left" width = "300" alt = "Profile photo of Beatriz Milz" caption = "<center><strong>Beatriz Milz</strong></center>">}}
+
+My name is Beatriz Milz, and I am an R-Lady from São Paulo - Brazil. I am currently doing a Ph.D. in Environmental Science and also work as a Data Scientist and an R teacher at Curso-R. 
+
+I am a tidyverse instructor certified by RStudio, and a co-organizer of communities and events, such as R-Ladies São Paulo, Latin-R, and satRday São Paulo. I also [have a blog](https://beamilz.com/) where I write about R:
```
Comment (beatrizmilz) [inline content/blog/2023-05-23-champions-program-teams/index.md:41]:
```suggestion
I am a tidyverse instructor certified by RStudio, and a co-organizer of communities and events, such as [R-Ladies São Paulo](https://rladies-sp.org/), [Latin-R](https://latin-r.com/), and [satRday São Paulo](https://saopaulo2019.satrdays.org/). I also [have a blog](https://beamilz.com/) where I write about R.
```

File: content/blog/2023-05-23-champions-program-teams/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+*In this series of blog posts, we introduce you to the ten teams of this first cohort and what they will be working on in the program.*
+
+Meet **Haydee Svab** and **Beatriz Milz** both from Brazil!
+
+
+## Haydee Svab - Champion
+
+{{< figure src = "/img/team/haydee-svab-mentee.jpeg" class = "pull-left" width = "300" alt = "Profile photo of Haydee Svab" caption = "<center><strong>Haydee Svab</strong></center>">}}
+
+I'm Haydee Svab, graduated in civil engineering, master of science - speciality transport and planning engineering. For the last few years I've been working as a researcher and consultant in my own company, ASK-AR. Our main projects deal with data analysis, smart cities, urban mobility and gender perspective. I was born and have always lived in Brazil, more specifically in São Paulo, where I founded the local RLadies chapter together with other amazing women.
```
Comment (beatrizmilz) [inline content/blog/2023-05-23-champions-program-teams/index.md:25]:
```suggestion
I'm Haydee Svab, graduated in civil engineering, master of science - speciality transport and planning engineering. For the last few years I've been working as a researcher and consultant in my own company, ASK-AR. Our main projects deal with data analysis, smart cities, urban mobility and gender perspective. I was born and have always lived in Brazil, more specifically in São Paulo, where I founded the [local RLadies chapter](https://www.meetup.com/rladies-sao-paulo/) together with other amazing women. 
```

**Review by beatrizmilz — COMMENTED** [review]
Hi @yabellini and @hsvab ! I added some links as a suggestion :) Thanks!

**Comment by yabellini** [comment]
@steffilazerte this is ready for review.

File: content/blog/2023-05-23-champions-program-teams/index.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+title: "rOpenSci Champions Program Teams: Meet Haydee Svab and Beatriz Milz"
+author:
+  - Haydee Svab
+  - Beatriz Milz
```
Comment (steffilazerte) [inline content/blog/2023-05-23-champions-program-teams/index.md:5]: Need author files

File: content/blog/2023-05-23-champions-program-teams/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+date: '2023-05-23'
```
Comment (steffilazerte) [inline content/blog/2023-05-23-champions-program-teams/index.md:6]: Reminder to tweak date and folder name as needed

File: content/blog/2023-05-23-champions-program-teams/index.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+{{< figure src = "/img/team/beatriz-milz-mentor.jpeg" class = "pull-left" width = "300" alt = "Profile photo of Beatriz Milz" caption = "<center><strong>Beatriz Milz</strong></center>">}}
+
+My name is Beatriz Milz, and I am an R-Lady from São Paulo - Brazil. I am currently doing a Ph.D. in Environmental Science and also work as a Data Scientist and an R teacher at Curso-R. 
+
+I am a tidyverse instructor certified by RStudio, and a co-organizer of communities and events, such as [R-Ladies São Paulo](https://rladies-sp.org/), [Latin-R](https://latin-r.com/), and [satRday São Paulo](https://saopaulo2019.satrdays.org/). I also [have a blog](https://beamilz.com/) where I write about R.
+
+I will be a mentor in this edition of the Champions program, and I hope to help 
+participants to create R packages and also review them for rOpenSci!
+
+_We're excited to have both Haydee and Bea be part of the Champions program. We are very exited for the progress they have in the project. Stay-tuned for the next series of introductions._
```
Comment (steffilazerte) [inline content/blog/2023-05-23-champions-program-teams/index.md:46]:
```suggestion
_We're excited to have both Haydee and Bea be part of the Champions program. We are looking forward to seeing how this project progresses. Stay tuned for the next series of introductions._
```

File: content/blog/2023-05-23-champions-program-teams/index.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+*In this series of blog posts, we introduce you to the ten teams of this first cohort and what they will be working on in the program.*
+
+Meet **Haydee Svab** and **Beatriz Milz** both from Brazil!
+
+
+## Haydee Svab - Champion
+
+{{< figure src = "/img/team/haydee-svab-mentee.jpeg" class = "pull-left" width = "300" alt = "Profile photo of Haydee Svab" caption = "<center><strong>Haydee Svab</strong></center>">}}
+
+I'm Haydee Svab, graduated in civil engineering, master of science - speciality transport and planning engineering. For the last few years I've been working as a researcher and consultant in my own company, ASK-AR. Our main projects deal with data analysis, smart cities, urban mobility and gender perspective. I was born and have always lived in Brazil, more specifically in São Paulo, where I founded the [local RLadies chapter](https://www.meetup.com/rladies-sao-paulo/) together with other amazing women.
```
Comment (steffilazerte) [inline content/blog/2023-05-23-champions-program-teams/index.md:25]:
```suggestion
I'm Haydee Svab, graduated in civil engineering, master of science - specialty transport and planning engineering. For the last few years I've been working as a researcher and consultant in my own company, ASK-AR. Our main projects deal with data analysis, smart cities, urban mobility and gender perspective. I was born and have always lived in Brazil, more specifically in São Paulo, where I founded the [local R-Ladies chapter](https://www.meetup.com/rladies-sao-paulo/) together with other amazing women. 
```

File: content/blog/2023-05-23-champions-program-teams/index.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+I'm Haydee Svab, graduated in civil engineering, master of science - speciality transport and planning engineering. For the last few years I've been working as a researcher and consultant in my own company, ASK-AR. Our main projects deal with data analysis, smart cities, urban mobility and gender perspective. I was born and have always lived in Brazil, more specifically in São Paulo, where I founded the [local RLadies chapter](https://www.meetup.com/rladies-sao-paulo/) together with other amazing women. 
+
+Despite loving and  professionally working with the R language, I don't know how to make packages - and I always wanted to make one, with data and model derived from my master's dissertation. So, I saw in the Champion Program the opportunity to make that wish come true. 
+
+Additionally, within the context of rOpenSci community, I recognize that the technical ability to create packages and its daily use can induce and facilitate the reuse and diffusion of codes from projects and initiatives that I am part of, be it R-Ladies or professional consultancy projects. Those will for sure benefit the R community, that will be able to leverage such packages.
```
Comment (steffilazerte) [inline content/blog/2023-05-23-champions-program-teams/index.md:29]:
```suggestion
Additionally, within the context of rOpenSci community, I recognize that the technical ability to create packages and their daily use can induce and facilitate the reuse and diffusion of codes from other projects and initiatives that I am part of, be it R-Ladies or professional consultancy projects. Those will for sure benefit the R community, which will be able to leverage such packages.
```

File: content/blog/2023-05-23-champions-program-teams/index.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+Finally, I am super excited for the learning period we will have and to get to know better and contribute to the rOpenSci community.
```
Comment (steffilazerte) [inline content/blog/2023-05-23-champions-program-teams/index.md:31]:
```suggestion
Finally, I am super excited for the learning period we will have and to better get to know and contribute to the rOpenSci community.
```

File: content/blog/2023-05-23-champions-program-teams/index.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+
+</br>
+
+## Beatriz Milz - Mentor
+
+{{< figure src = "/img/team/beatriz-milz-mentor.jpeg" class = "pull-left" width = "300" alt = "Profile photo of Beatriz Milz" caption = "<center><strong>Beatriz Milz</strong></center>">}}
+
+My name is Beatriz Milz, and I am an R-Lady from São Paulo - Brazil. I am currently doing a Ph.D. in Environmental Science and also work as a Data Scientist and an R teacher at Curso-R. 
+
+I am a tidyverse instructor certified by RStudio, and a co-organizer of communities and events, such as [R-Ladies São Paulo](https://rladies-sp.org/), [Latin-R](https://latin-r.com/), and [satRday São Paulo](https://saopaulo2019.satrdays.org/). I also [have a blog](https://beamilz.com/) where I write about R.
```
Comment (steffilazerte) [inline content/blog/2023-05-23-champions-program-teams/index.md:41]:
```suggestion
I am a tidyverse instructor certified by Posit, and a co-organizer of communities and events, such as [R-Ladies São Paulo](https://rladies-sp.org/), [Latin-R](https://latin-r.com/), and [satRday São Paulo](https://saopaulo2019.satrdays.org/). I also [have a blog](https://beamilz.com/) where I write about R.
```

**Review by steffilazerte — COMMENTED** [review]
As with the others, just a couple of minor suggestions and we need author files, then we're good to go!
* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok - CHECK
* [X] alternative text of images is informative
* [ ] author metadata is provided with correct folder name -> **Need author files**
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary

**Comment by yabellini** [comment]
Hi @beatrizmilz and @hsvab we need you create an author profile.  Here are the instructions to create one: https://blogguide.ropensci.org/authortechnical.html#createauthorfile
Please let me know if you have any questions!

**Comment by beatrizmilz** [comment]
> Hi @beatrizmilz and @hsvab we need you create an author profile. Here are the instructions to create one: https://blogguide.ropensci.org/authortechnical.html#createauthorfile Please let me know if you have any questions!

Here it is: https://github.com/ropensci/roweb3/pull/560

**Comment by beatrizmilz** [comment]
https://ropensci.org/blog/2023/05/09/ropensci-champions-program-teams-meet-haydee-svab-and-beatriz-milz/ it's online! Thanks !

**Comment by yabellini** [comment]
Thanks to you!

**Pull request merged**